quickstarts/aws/aws-iot-twinmaker/config.yml

```diff
@@ -0,0 +1,33 @@
+slug: amazon-iot-twinmaker
+description: |-
+  ## What is AWS IOT Twinmaker?
+  AWS IoT TwinMaker is an AWS IoT service that you can use to build operational digital twins of physical and digital systems. AWS IoT TwinMaker creates digital visualizations using measurements and analysis from a variety of real-world sensors, cameras, and enterprise applications to help you keep track of your physical factory, building, or industrial plant.
+  ### Get started!
+  Start monitoring AWS IOT Twinmaker by connecting Amazon Web Services (AWS) to New Relic! Take a look at our documentation for information on how to set up this integration.
+  ### More info
+  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-iot-twinmaker-monitoring-integration/) to learn more about New Relic monitoring for Amazon Timestream.
+summary: |-
+  Monitor AWS IOT Twinmaker by connecting AWS to New Relic
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+title: AWS IoT Twinmaker
+documentation:
+  - name: AWS IOT Twinmaker installation docs
+    description: |
+      Monitor AWS IoT Twinmaker by connecting AWS to New Relic.
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-iot-twinmaker-monitoring-integration/
+keywords:
+  - aws
+  - amazon web services
+  - aws iot
+  - aws iot twinmaker
+  - twinmaker
+dashboards:
+  - aws-twinmaker
+dataSourceIds:
+  - amazon-cloudwatch-metric-streams
+alertPolicies:
+  - aws-twinmaker
```

Comment:
Same problem here as above

Reply:
Thanks @sarahkitten, I made the required changes.